### ISSUE TITLE
[Suggestion] Re-instate a new 'Status' page with additional button features

### ISSUE THREAD

**Issue by TheMax99** [segler-alex/RadioDroid#325]
Current version seems to cut off text of track/artist information in the 'now playing' box at bottom of window for longer titles.

Suggestion:
Instead of press on main bar of the bottom 'now playing' box instigates 'Arm Record' function, instead I suggest press on 'now playing' box opens page similar to the old 'Status' page, so this page will display full station and track/artist info and also three buttons for 'stop and close player', 'Record instantly', and 'Make screen grab' (to store track/artist info for future reference).

Also maybe instant navigation to the 'Recordings' folder to allow delete/move of recordings in default file browser?

'Return' navigates back to last page.

**Comment by Wikinaut** [comment]
Duoplicate to #278 . Please add your comments there.

**Comment by TheMax99** [comment]
Not a duplicate surely because the main part of the request is to add several new features, not just the 'clipped text' issue?

**Comment by Wikinaut** [comment]
@TheMax99 Yep, you are right. I am also in favour of having the old status page back.
